test/snap-and-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd dex-contracts/
```

Comment:
Could this script and the functionality of revert-cleanup be exposed as functions in a utility script instead of creating multiple files which are automatically executed when sourced?

Reply:
Not sure I understand. Are you referring to the `ganache/makeSnapshot` and `ganache/revertSnapshot`  being in the same file? This was the case at one point, but @josojo mentioned a reason I can't recall about making them in two two different ones. Maybe we can all put some ideas together here and come up with something great! I have been tinkering with your suggestion to do something like this


```js
// make ganache snapshot 
const makeSnapshot = async function(web3Provider=web3) {
  const res = await send("evm_snapshot", web3Provider)
  return res
}

const revertSnapshot = async function(snapID, web3Provider=web3) {
  await send("evm_revert", [snapID], web3Provider)
}
```